delete.go

```diff
@@ -2,19 +2,11 @@ package buildah
 
 import (
 	"fmt"
-	"os"
 )
 
 // Delete removes the working container.  The Builder object should not be used
```

Comment:
Lets discuss this one outside of this pull request.  I think Nalin is referring to the Buildah object in the code.